socorro/processor/processor_app.py

```diff
@@ -144,10 +133,6 @@
             'filter_on_legacy_processing': True,
             'routing_key': 'socorro.normal',
         },
-
-        'signature': {
-            'collapse_arguments': True,
-        },
```

Comment:
I changed signature generation to not use config at all, so this has been orphaned for months and can be removed.